services/user.py

```diff
@@ -0,0 +1,45 @@
+from db.models import User
+
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str, password: str, email: str = None,
+        first_name: str = None, last_name: str = None
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email or "",
+        first_name=first_name or "",
+        last_name=last_name or "",

```

Comment:
strange approach. and when some information was not passed it should be None not ""

Reply:
Thanks, changed.